language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ModulePartNodeContext.java

```diff
@@ -57,7 +63,35 @@ public ModulePartNodeContext() {
         completionItems.addAll(addTopLevelItems(context));
         completionItems.addAll(this.getTypeItems(context));
         completionItems.addAll(this.getPackagesCompletionItems(context));
+        this.sort(context, node, completionItems);
 
         return completionItems;
     }
+
+    @Override
+    public void sort(LSContext context, ModulePartNode node, List<LSCompletionItem> items, Object... metaData) {
+        for (LSCompletionItem item : items) {
+            CompletionItem cItem = item.getCompletionItem();
+            if (item instanceof SnippetCompletionItem
+                    && (((SnippetCompletionItem) item).kind() == SnippetBlock.Kind.SNIPPET
+                    || ((SnippetCompletionItem) item).kind() == SnippetBlock.Kind.STATEMENT)) {
```

Comment:
Shall we move this check into a util function in `SortingUtil` such as `SortingUtil.isStatementCompletionItem(item)`

Reply:
Sounds good, will do.